doc/loganalyzer_to_pytest.md

```diff
@@ -0,0 +1,74 @@
+#### Requirements
+Move existed loganalyzer to be supported by pytest framework.
+
+#### Overview
+At this approach all the work can be done on the Ansible host if script will download extracted syslog file to the Ansible host.
+
+Current loganalyzer implementation description:
+https://github.com/Azure/SONiC/wiki/LogAnalyzer
+
+#### Development
+Module "loganalyzer.py" with class "Loganalyzer". Class is intendent to do:
+- Add start marker to the DUT syslog
+- Extract DUT syslog based on the markers. Download extracted logs to the Ansible host
+- Perform analize of downloaded syslog and return the result
+
+- Configure regular expressions which will be used to found matches in the logs
+
+"Loganalyzer" class interface:
+- __init__(ansible_host: ansible_host)
+- load_common_config() - Load regular expressions from common configuration files: match, expect and ignore which are located in some configuration directory or with current module. Clear previous configured match, expect and ignore. Save loaded configuration.
+- parse_regexp_file(file_path) - read and parse regular expressions from specified file. Return list of strings of defined regular expressions.
+- run_cmd(callable, *args, **kwargs) - call function and analyze DUT logs, return the same result as "analyze" function
+- init() - Add start marker to the DUT syslog.
+- analyze() - Extract syslog and copy it to ansible host. Analyze extracted syslog files localy. Return python dictionary object.
+Return example:
+{"counters": {"match": 1, "expected_match": 0, "expected_missing_match": 0},
+ "match_files": {"/tmp/syslog": {"match": 0, "expected_match": 32},
+					"/tmp/syslog1": {"match": 0, "expected_match": 15}
+						},
+ "match_messages": {"/tmp/syslog1": ["Message 1", "Message 2", "Message n"],
+						 "/tmp/syslog2": ["Message 1", "Message 2", "Message n"]
+						 }
+}
+- save_full_log(file_path) - save downloaded DUT syslog logs to the Ansible host folder specified in 'path' input parameter.

```

Comment:
Save log to DUT or to sonic-mgmt container?

Reply:
To sonic-mgmt container

Reply:
save_full_log(file_path) - will be better if this will download /tmp/syslog from the DUT to the Ansible host. Because temp logs which are composed by analyze method will be deleted.